com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDTarget.cs

```diff
@@ -80,6 +80,8 @@ public override bool IsNodeAllowedByTarget(Type nodeType)
         public HDTarget()
         {
             displayName = "HDRP";
+            supportsExposableProperties = true;
```

Comment:
If I'm reading this correctly, this property just causes the inspector to reserve a column for the exposed checkboxes?

Reply:
exactly